tests/Routing/RoutingRouteTest.php

```diff
@@ -211,6 +213,24 @@ public function testControllerClosureMiddleware()
         );
     }
 
+    public function testHandler()
+    {
+        $router = $this->getRouter();
+        $router->get('foo/bar', [
+            'uses' => function () {
+                return 'hello';
+            },
+            'handler' => 'Illuminate\Tests\Routing\RouteTestHandler',
+        ]);
+
+        $router->dispatch(Request::create('foo/bar', 'GET'))->getContent();
```

Comment:
Shouldn't this assert that the content equals `handled` and not `hello`?

Reply:
I'm not sure. but ExceptionHandler are managed in Http layer. 
Here it's not triggered because we are working low-level with routes.

The only test I could do, is to check ServiceContainer.

Reply:
> Here it's not triggered because we are working low-level with routes.

You can convert this into Integration tests (instead of unit test) and make a proper request.

Reply:
Yes, but I think this is enough for this feature. 
I don't want to test if Laravel ExceptionHandler current logic works, there should be some other test to check it.